### PR TITLE
Create asheer.json

### PR DESCRIPTION
**Pull request by NullyIsHere**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
[altex.page](altex.page)

PD: Fixed things asked.

### CODE REVIEW

**Comment by NullyIsHere** [comment]
URL preview isn't valid, use
[altex.page](https://altex.page/?from=isadev)

**Review by DEV-DIBSTER — APPROVED** [review]

File: domains/asheer.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "NullyIsHere",
+    "email": "",
```
Comment (DEV-DIBSTER) [inline domains/asheer.json:4]: Email is neeed.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/asheer.json (hunk 1 above)
Comment (NullyIsHere) [reply domains/asheer.json:4]: https://github.com/is-a-dev/register/pull/14374
Stef told me email wasnt neccesary.

File: domains/asheer.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "discord": "itsasheer",
```
Comment (CuteDog5695) [inline domains/asheer.json:5]:
```suggestion
    "discord": "itsasheer"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by NullyIsHere** [comment]
Committed what you asked.

**Review by iostpa — APPROVED** [review]
Seems fine to me now (If URL record doesn't redirect to your website then you can use https://redirect.pizza/)

**Pull request merged**